notebooks/22.Unsupervised_learning-anomaly_detection.ipynb

```diff
@@ -84,7 +84,7 @@
     "from sklearn.datasets import make_blobs\n",
     "\n",
     "X, y = make_blobs(n_features=2, centers=3, n_samples=500,\n",
-    "                  random_state=666)"
```

Comment:
Any reason for that? I feel like random states don't need to be consistent, and from the code it's impossible to see if the result of the example is the same.

Reply:
I commented on that in the particular commit, now realized that might not be the best place.
Well make_blobs with 3 centers and a seed of 666 produces two blobs that overlap (a clear sign that 666 is pure evil), which is fine if that is your goal but visually looks a bit weird for this demonstration.